source/partials/release_notes/_release-17-10-0.md.erb

```diff
@@ -0,0 +1,9 @@
+**Important:** On Sep 11th, we made an announcement asking users to hold off upgrading to 17.9.0 because of issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>). This release provides fix for the same. While the issue affects a small number of users, we think it still warrants a new release.
```

Comment:
**Important:** After the 17.9 release, we made an announcement asking users to hold off upgrading their GoCD instance. We found an issue (<%= link_to("#3834", "https://github.com/gocd/gocd/issues/#{3834}") %>) affecting a small set of users. This release provides a fix for the same.